### PR TITLE
Fix/scout

### PR DESCRIPTION
**Pull request by igor-kamil**
fix scout returning empty collection

### CODE REVIEW

File: app/Providers/TelescopeServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    /**
+     * Register the Telescope gate.
+     *
+     * This gate determines who can access Telescope in non-local environments.
+     *
+     * @return void
+     */
+    protected function gate()
```
Comment (eronisko) [inline app/Providers/TelescopeServiceProvider.php:63]: How are you actually planning to guard this?

File: app/Providers/TelescopeServiceProvider.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    {
+        // Telescope::night();
+
+        $this->hideSensitiveRequestDetails();
+
+        Telescope::filter(function (IncomingEntry $entry) {
+            if ($this->app->environment('local')) {
+                return true;
+            }
+
```
Comment (eronisko) [inline app/Providers/TelescopeServiceProvider.php:27]: Note that right now you'll be only tracking errors outside of local

**Review by eronisko — COMMENTED** [review]
Haven't tested it but the code looks OK... Did you mean to include Telescope as well?

**Review by rastislav-chynoransky — APPROVED** [review]
I would also delete all the unnecessary classes/configs/env vars that came with https://github.com/SlovakNationalGallery/exlibris.pamatniknarodnihopisemnictvi.cz/pull/6, otherwise LGTM!

**Pull request merged**